invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -37,20 +38,24 @@ class MARCXMLSchema(Schema):
     sources = fields.Method(
         "get_sources"
     )  # Corresponds to references in the metadata schema
-    coverage = fields.Method(
+    locations = fields.Method(
         "get_locations"
     )  # Corresponds to locations in the metadata schema
     formats = fields.Method("get_formats")
-    parent_id = fields.Method("get_id")  # No GOOD
+    parent_id = fields.Method("get_id")
     community_id = fields.Method("get_id_community")
     last_updated = fields.Method("last_updated")
     sizes = fields.Method("get_sizes")
     version = fields.Method("get_version")
     funding = fields.Method("get_funding")
 
-    def get_version(self, obj):
+    @staticmethod
+    def get_version(obj):
         """Get the version of the record."""
-        return obj["metadata"].get("version", missing)
+        version = obj.get("versions", {})
```

Comment:
I think this is the revision id (internal), while `metadata.version` is what we want to serialize? (i.e. version given in the deposit form by the user.

Reply:
That's what it was like before, for some reason I thought this was an error :sweat_smile: 
I'll put it back the way it was :)